staging/prometheus-operator/templates/mesosphere-hooks/prometheus-cluster-id-hooks.yaml

```diff
@@ -67,4 +80,29 @@ spec:
             - -c
             - kubectl create configmap {{ .Values.mesosphereResources.hooks.prometheus.configmapName }} -n {{ .Release.Namespace }} -oyaml --dry-run --save-config --from-literal=CLUSTER_ID=$(kubectl get namespace kube-system -o jsonpath="{.metadata.uid}") | kubectl apply -f -
       restartPolicy: OnFailure
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cleanup-{{ .Values.mesosphereResources.hooks.prometheus.configmapName }}
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    helm.sh/hook: pre-delete
+    helm.sh/hook-weight: "3"
+    helm.sh/hook-delete-policy: hook-succeeded,before-hook-creation
+spec:
+  template:
+    metadata:
+      name: cleanup-{{ .Values.mesosphereResources.hooks.prometheus.configmapName }}
+    spec:
+      serviceAccountName: {{ template "prometheus-operator.prometheus.serviceAccountName" . }}
+      containers:
+        - name: kubectl
+          image: bitnami/kubectl:1.16.2
```

Comment:
looks like we are using `kubectl` at multiple places. We should then templatize the image then

Reply:
Yes, it would be helpful to have one topical place to configure the kubectl version we use.